apps/main/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 5.2.1 on 2025-05-19 20:48
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Profile',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('bio', models.TextField(blank=True, max_length=50)),
+                ('profile_picture', models.ImageField(blank=True, upload_to='profile_pics/')),
+                ('location', models.CharField(blank=True, max_length=30)),
```

Comment:
What location is referred to? Is this a country or full address? If we talk about a country, then I'd recommend using country codes. Read up on country codes. For them, you need to use enumerations to prevent random values from being set. Read about country codes, enumerations in Python.